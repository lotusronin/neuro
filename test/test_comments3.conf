conf = {
'src' : 'test_comments3.nro',
'expected' : 'test_comments3.expected'
}
