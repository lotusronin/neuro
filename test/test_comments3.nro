import stdio;

fn main() : u32 {
    stdio::puts("before block comment");
    /*
     * //stdio::puts("in block comment");
     * //stdio::puts("in block comment2");
     */
    stdio::puts("after block comment");
    return 0;
}
