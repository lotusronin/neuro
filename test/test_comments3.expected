before block comment
after block comment
